plugins/rapid7_insightidr/help.md

```diff
@@ -959,39 +959,41 @@ Example output:
 ```
 {
   "log": {
-    "id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f",
-    "links": [
-      {
-        "href": "https://example.com",
-        "rel": "Related"
-      }
-    ],
-    "logsets_info": [
-      {
-        "id": "bc38a911-65f1-4755-cca3-a330a6336b3a",
-        "links": [
-          {
-            "href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85",
-            "rel": "Self"
-          }
-        ],
-        "name": "Unparsed Data",
-        "rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a"
+    "log": {
+      "id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f",
+      "links": [
+        {
+          "href": "https://example.com",
+          "rel": "Related"
+        }
+      ],
+      "logsets_info": [
+        {
+          "id": "bc38a911-65f1-4755-cca3-a330a6336b3a",
+          "links": [
+            {
+              "href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85",
+              "rel": "Self"
+            }
+          ],
+          "name": "Unparsed Data",
+          "rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a"
+        }
+      ],
+      "name": "Windows Defender",
+      "retention_period": "default",
+      "rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:log:bc38a911-65f1-4755-cca3-a330a6336b3a",
+      "source_type": "token",
+      "structures": [
+        "1238a911-65f1-4755-cca3-a330a6336b3a"
+      ],
+      "token_seed": null,
+      "tokens": [
+        "bc38a911-65f1-4755-cca3-a330a6336b3a"
+      ],
+      "user_data": {
+        "platform_managed": "true"
```

Comment:
Probably it comes from `plugin.spec.yaml` file (the doubled nested output key). Could you please fix that `example` in `plugin.spec.yaml` file? Thanks!

Reply:
I don't see it nested in the example: this would be line 1877 and 1887 in the spec file accordingly
`        example: {"log": {"id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f","name": "Windows Defender","tokens": ["bc38a911-65f1-4755-cca3-a330a6336b3a"],"structures": ["1238a911-65f1-4755-cca3-a330a6336b3a"],"user_data": {"platform_managed": "true"},"source_type": "token","token_seed": null,"retention_period": "default","links": [{"rel": "Related","href": "https://example.com"}],"rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:log:bc38a911-65f1-4755-cca3-a330a6336b3a","logsets_info": [{"id": "bc38a911-65f1-4755-cca3-a330a6336b3a","name": "Unparsed Data","rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a","links": [{"rel": "Self","href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85"}]}]}}
`

`        example: {"logs": [{"log": {"id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f","name": "Windows Defender","tokens": ["bc38a911-65f1-4755-cca3-a330a6336b3a"],"structures": ["1238a911-65f1-4755-cca3-a330a6336b3a"],"user_data": {"platform_managed": "true"},"source_type": "token","token_seed": null,"retention_period": "default","links": [{"rel": "Related","href": "https://example.com"}],"rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:log:bc38a911-65f1-4755-cca3-a330a6336b3a","logsets_info": [{"id": "bc38a911-65f1-4755-cca3-a330a6336b3a","name": "Unparsed Data","rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a","links": [{"rel": "Self","href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85"}]}]}}]}
`

Reply:
@ewilson-r7 


for the example 

```
example: {"log": {"id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f","name": "Windows Defender","tokens": ["bc38a911-65f1-4755-cca3-a330a6336b3a"],"structures": ["1238a911-65f1-4755-cca3-a330a6336b3a"],"user_data": {"platform_managed": "true"},"source_type": "token","token_seed": null,"retention_period": "default","links": [{"rel": "Related","href": "https://example.com"}],"rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:log:bc38a911-65f1-4755-cca3-a330a6336b3a","logsets_info": [{"id": "bc38a911-65f1-4755-cca3-a330a6336b3a","name": "Unparsed Data","rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a","links": [{"rel": "Self","href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85"}]}]}}
```

it needs to be 

```
example: {"id": "0b9a242d-d2fb-4e42-8656-eb5ff64d652f","name": "Windows Defender","tokens": ["bc38a911-65f1-4755-cca3-a330a6336b3a"],"structures": ["1238a911-65f1-4755-cca3-a330a6336b3a"],"user_data": {"platform_managed": "true"},"source_type": "token","token_seed": null,"retention_period": "default","links": [{"rel": "Related","href": "https://example.com"}],"rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:log:bc38a911-65f1-4755-cca3-a330a6336b3a","logsets_info": [{"id": "bc38a911-65f1-4755-cca3-a330a6336b3a","name": "Unparsed Data","rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a","links": [{"rel": "Self","href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85"}]}]}
```


this is due to the Output being treated as object  where each of the titles will be treated as a key, eg the example filed that added here will be added to the log filed and not be the whole example out put 

```    
output:
      log:
        title: Log
        description: Requested log
        type: logsets_info
        required: true
        example: <above>
```

Reply:
I understand. I've implemented this change before but it broke the Output validator so I'd appreciate if someone on the team could take a look at this since this change isn't part of the PR.

Reply:
Fix has been applied to help.md json formatting for Get a Log and Get All Logs sections.